apps/playwright/tests/applicant/application.spec.ts

```diff
@@ -286,9 +286,5 @@ test.describe("Application", () => {
         level: 2,
       }),
     ).toBeVisible();
-
```

Comment:
Hmm, I did since it was flaky (not always a draft) and the line above does a similar check so I just wanted to make playwright consistent. 

However, I can add it back if we want since it is technically not even part of the scope of this ticket.

Reply:
That's cool.  Just making sure it wasn't a merge artifact or something.